test/conftest.py

```diff
@@ -107,13 +116,23 @@ def pytest_collection_modifyitems(config, items):
         # if the cli argument "--document_store_type" is used, we want to skip all tests that have markers of other docstores
         # Example: pytest -v test_document_store.py --document_store_type="memory" => skip all tests marked with "elasticsearch"
         document_store_types_to_run = config.getoption("--document_store_type")
+        document_store_types_to_run = document_store_types_to_run.split(",")
         keywords = []
+
+        if "milvus" in document_store_types_to_run and os.getenv("MILVUS2_ENABLED"):
+            document_store_types_to_run.remove("milvus")
+            document_store_types_to_run.append("milvus2")
```

Comment:
I see also below that in many tests you distinguish `milvus` and `milvus2` fixtures as if they could run together. I think there's no setup where I could do something like `--document_store_types="milvus,milvus2"`: therefore, why distinguish between them?

I tried to find the reason in the diff below but I could not find a clear reason. Let me know if I missed it :slightly_smiling_face:

Reply:
I see! It makes sense then. I'll make sure in the CI the proper tests are run.